cql3/statements/describe_statement.cc

```diff
@@ -244,11 +252,27 @@ description index(const data_dictionary::database& db, const sstring& ks, const
     return description(db.real_database(), **idx, with_internals);
 }
 
+std::optional<description> describe_cdc_log_table(const data_dictionary::database& db, const sstring& ks, const sstring& base_name) {
```

Comment:
You did not put this function into a header, so I think it should be `static`.

Reply:
It is possible to create a table `t` (not cdc-enabled) and then `t_scylla_cdc_log`. The function should not be used to generate the `ALTER` statement for `t_scylla_cdc_log` because it would not be correct.

I don't think this problem occurs in the current implementation, but you should either:

- Add a comment explaining the precondition ("base_name must be a name of a cdc-enabled table)
- Check that base_name is cdc-enabled and throw if it isn't

Reply:
> You did not put this function into a header, so I think it should be `static`.

The function is in a anonymous namespace, isn't it enough?

Reply:
It's enough. I forgot that the possibility of putting an item into an anonymous namespace exists, I only looked at the local diff so I didn't notice the `namespace`.